sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Client/CryptographyClient.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Security.KeyVault.Cryptography.Client
+{
+    using Azure.Core;
+    using Azure.Core.Pipeline;
+    using Azure.Security.KeyVault.Cryptography.Base;
+    using Azure.Security.KeyVault.Cryptography.CryptoProvider;
+    using Azure.Security.KeyVault.Cryptography.Interface;
+    using Azure.Security.KeyVault.Cryptography.Utilities;
+    using Azure.Security.KeyVault.Keys;
+    using System;
+
+    /// <summary>
+    /// The Cryptography Client 
+    /// </summary>
+    public class CryptographyClient
+    {
+        #region const
+        const string DEFAULT_KV_SCOPE_URI = @"https://vault.azure.net/.default";
+        #endregion
+
+        #region fields
+
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public TokenCredential Credentials { get; }
+        /// <summary>
+        /// Cryptography Client Options
+        /// </summary>
+        public CryptographyClientOptions CryptoClientOptions { get; }
+
+        /// <summary>
+        /// HttpPipeline
+        /// </summary>
+        public HttpPipeline Pipeline {get;}
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public ICryptographyProvider CryptoProvider { get; protected set; }
```

Comment:
shouldn't expose this

Reply:
ah I remember, we want this to be private.
Will make it internal